keyloop/api/v1/exceptions.py

```diff
@@ -1,3 +1,8 @@
 class IdentityNotFound(Exception):
     """Raise an error if identity has not been found"""
     pass
+
+
+class AuthSessionUnauthorized(Exception):
```

Comment:
Acho que podemos melhorar o nome. Auth aparece duas vezes e lendo fico na dúvida se são a mesma coisa.
```suggestion
class AuthenticationFailed(Exception):
```